docs/Installation Guide.md

```diff
@@ -0,0 +1,319 @@
+
+The VirtualBox and Vagrant installers can be downloaded from the following websites:
+
+1. [VirtualBox](https://www.virtualbox.org/wiki/Downloads)
+2. [Vagrant](http://www.vagrantup.com/downloads.html)
+
+Install VirtualBox and Vagrant by running the installers.
+
+## Download PeaceCorps Ubuntu Virtual Machine
+
+A Vagrant file is located in the top level directory for the project (at [https://github.com/Nerdylicious/app-web-server/blob/master/Vagrantfile](https://github.com/Nerdylicious/app-web-server/blob/master/Vagrantfile) found on GitHub. In case you do not have a copy of this Vagrant file, here are it's contents:
+```
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.provider :virtualbox do |vb|
+    vb.gui=true
+  end
+  config.vm.box = "pc-web-dev"
+
+  # config.vm.box_url = "../box/pc-web-05192014-i386.box"
+  config.vm.box_url = "http://54.183.32.240/vagrant/box/pc-web-05192014-i386.box"
+
+Vagrant.configure("2") do |config|
+  config.vm.network "forwarded_port", guest: 80, host: 8080,
+  auto_correct: true
+end
+  config.vm.network "forwarded_port", guest: 8000, host: 8001
+  config.vm.network "private_network", ip: "192.168.33.10"
+  config.vm.network "public_network"
+
+end
+```
+Save this file as **Vagrantfile** (if you don't already have this file) in the top level directory for the project.
+
+To download the VM, run the following command:
+
+    vagrant up
+
+You must wait a few minutes for the VM to be downloaded completely.
+
+## Using Vagrant
+
+The `vagrant up` command also boots the Virtual Machine.
+
+Once the VM download has completed, upon boot, it may ask you to choose an `Available bridged network interface`. The first option wil work in most cases.
+
+You may come across a message that says `default: Warning: Remote connection disconnect. Retrying...` This message means that the VM is still booting up which is why we cannot establish a connection with it. It is normal to wait on this message for a few minutes (~5 minutes in my case) before we are able to get a connection to the VM. You may need to wait a few minutes until you get a message saying `default: Machine booted and ready!`.
+
+File syncing will work properly after you receive this message: `default: Mounting shared folders`. Please wait for this message before proceeding to the next steps.
+
+Once the VM booted up (and you were able to receive the messages specified above), you can ssh onto the VM by running the command:
+
+    vagrant ssh
+
+You will notice that the project is now synced to this VM by changing directory to **/vagrant** in the virtual machine.
+
+    cd /vagrant
+
+When you make any changes to the project locally, these changes are also reflected (synced) to the project files located in **/vagrant**, and vice versa.
+
+## Install Project Dependencies
```

Comment:
I am trying to understand if we can do better than asking user to install dependencies manually(basically automate it) 
Here are few suggestions. 
1) you can putt all these dependencies in a install_dependencies.sh script and then call that script from vagrant  sort of like this -->
    config.vm.provision :shell do |shell|
        shell.path = "install_dependencies.sh"
    end
2) Another option is  setup.py .. that is another place we can look into moving dependencies. 
http://www.scotttorborg.com/python-packaging/dependencies.html